tests/settings.py

```diff
@@ -27,6 +27,7 @@
     'django.contrib.staticfiles',
     'django.contrib.admin',
     'django_freeradius',
+    'rest_framework'
```

Comment:
you likely have to add DRF in requirements.txt too as in here:
https://github.com/netjson/django-netjsongraph/blob/master/requirements.txt#L2